nwbwidgets/base.py

```diff
@@ -96,9 +96,9 @@ def show_neurodata_base(
     return widgets.VBox(info + [accordion])
 
 
-def dict2accordion(
-    d: dict, neurodata_vis_spec: dict, **pass_kwargs
-) -> widgets.Accordion:
+def dict2accordion(d: dict, neurodata_vis_spec: dict, **pass_kwargs) -> widgets.Widget:
+    if len(d) == 1:
+        return nwb2widget(list(d.values())[0], neurodata_vis_spec=neurodata_vis_spec)
     children = [widgets.HTML("Rendering...") for _ in d]
     accordion = widgets.Accordion(children=children, selected_index=None)
     for i, label in enumerate(d):
```

Comment:
```suggestion
    for i, label in enumerate(d):
      if hasattr(d[label], "description") and d[label].description:
              accordion.set_title(i, label + ": " + d[label].description)
          else:
              accordion.set_title(i, label)
```
essentially line 109 should be removed. Github does not allow me to write suggestion on that line